### ISSUE TITLE
Question about mixloss

### ISSUE THREAD

**Issue by wangnaihao** [JackYFL/DISC#3]
您好！对于您的工作我非常感兴趣，我在再次阅读您论文和代码时发现论文中mixup用的是BCELoss，但是我发现代码中是普通的CELoss，请问是BCELoss不如CELoss，后续调整了吗？感谢您的答复!

**Comment by JackYFL** [comment]
DISC代码中针对weak set使用的是self.mixup_loss，这部分使用的是BCELoss；代码中针对Clean set使用的是CELoss，如果您有其他问题可以随时与我联系。

**Comment by wangnaihao** [comment]
您好，感谢您的回复，我理解self.mixup_loss是实例化loss包中mixup类，但是我没有找到bce部分的相关代码，请问是我漏看了哪里吗

**Comment by JackYFL** [comment]
这里的bce损失并没有调用已有的torch库函数，而是重新实现的：loss = -torch.mean(torch.sum(F.log_softmax(mixed_p, dim=1) * mixed_y, dim=1))。

**Comment by wangnaihao** [comment]
请问这个如何体现bce中的binary，这个形式我理解是普通的celoss，感谢您的回答！

**Comment by JackYFL** [comment]
这里应该是我的论文中写错了，代码中确实使用的是CEloss，非常感谢您指出这个问题~

**Comment by wangnaihao** [comment]
感谢您的回复。另外 我在尝试cifar10的60 idn噪音的情况下有些疑惑，我发现这个噪音相比其他比率的噪音有些特殊，我的效果学习的很差。我注意到您的代码里对60的比率做了特殊处理，请问60比率的噪音有什么特殊之处吗，或者您有什么优化的心得，感谢您的指导，这将对我帮助巨大！

**Comment by JackYFL** [comment]
根据实验，我觉得模型前期预热的时间可能对结果的影响比较大。
